pkg/controllers/management/node/controller.go

```diff
@@ -222,8 +224,6 @@ func (m *Lifecycle) getNodePool(nodePoolName string) (*v32.NodePool, error) {
 }
 
 func (m *Lifecycle) Remove(obj *v32.Node) (runtime.Object, error) {
```

Comment:
So I agree with your feedback, but IMO since this is a backport PR for https://github.com/rancher/rancher/pull/38850 which is already merged, I don't think that's a good idea. I generally try to keep backports as close to cherry-picking as possible, because in the future trying to cherry-pick again and again with minor changes sprinkled in becomes almost impossible. I'll stick to `apimgmtv3` in the future though.

Reply:
Since a new PR was needed for release/v2.7 I implemented this suggestion